test/common_utils.py

```diff
@@ -48,3 +47,7 @@ def random_int_tensor(seed, size, low=0, high=2 ** 32, a=22695477, c=1, m=2 ** 3
     """ Same as random_float_tensor but integers between [low, high)
     """
     return torch.floor(random_float_tensor(seed, size, a, c, m) * (high - low)) + low
+
+
+def _num_stft_bins(signal_len, fft_len, hop_length, pad):
```

Comment:
I'd move this into functional.py, especially if it starts with an underscore which indicates that its a local convenience function.

Reply:
It's only used for tests though, should it be in `test_transforms.py` then? Or rather, I should move all the contrib functional tests (and this helper) to `test_functional.py` right?